app/services/open_library_service.rb

```diff
@@ -0,0 +1,6 @@
+class OpenLibraryService
+  include HTTParty
+  def book_info(isbn)
+    HTTParty.get('https://openlibrary.org/api/books?bibkeys=ISBN:' + isbn + '&format=json&jscmd=data')
```

Comment:
Use the get method param for query params instead of defining them inside the url